mkdocs.yml

```
@@ -57,7 +57,8 @@ nav:
           - Rcon sample: samples/rcon.md
           - sACN:
                 - sACN Receiver sample: samples/sacn-receiver.md
-                - sACN Sender sample: samples/sacn-sender.md
+                - sACN Receiver sample: samples/sacn-sender.md
```

sACN Sender was correct here. The receiver is one line above and also both filenames are correct.
```suggestion
                - sACN Sender sample: samples/sacn-sender.md
```